src/devices/video/EmuNV2A_PGRAPH.cpp

```diff
@@ -4815,8 +4815,8 @@ int pgraph_handle_method(
                         if (method_count==2){
```

Comment:
the if (method_count==2){ in line 4815 could be removed as well, remember to remove the other "}" associated with this if statement as well.
there are only two vertex shader modes in PB, program and fixed mode. 
I would suggest to removed all unused code in this section and keep these two lines 
 g_NV2A_VertexShaderMode = VertexShaderMode::ShaderProgram;
g_NV2AVertexShader_dirty = true;